_pages/coc.md

```diff
@@ -85,14 +101,34 @@ We won't share your contact information or the contents of your message with any
 <input type="submit" value="Send">
 </form>
 
+## Processing Code of Conduct reports
```

Comment:
does this render properly? the text is italicized

Reply:
yup, I think it's the html or something
<img width="786" alt="screen shot 2017-12-31 at 7 46 27 pm" src="https://user-images.githubusercontent.com/5634097/34464900-526151ba-ee63-11e7-951a-82f038888430.png">